docs/install-deps

```diff
@@ -1 +1 @@
-conda install -y $(python -c "import yaml; f=open('$(dirname ${BASH_SOURCE[0]})/environment.yml'); print(' '.join(yaml.safe_load(f)['dependencies']))")
+conda install -q -y $(python -c "import yaml; f=open('$(dirname ${BASH_SOURCE[0]})/environment.yml'); print(' '.join(yaml.safe_load(f)['dependencies']))")
```

Comment:
`-q` suppresses a progress bar that looks messy in CI output. It was fine when this was just used on the command line by developers, but this will improve CI logs.